rustls/src/error.rs

```diff
@@ -305,6 +308,7 @@ impl fmt::Display for Error {
             }
             Error::CorruptMessage => write!(f, "received corrupt message"),
             Error::NoCertificatesPresented => write!(f, "peer sent no certificates"),
+            Error::UnsupportedNameType => write!(f, "presented server name wasn't valid"),
```

Comment:
I suggest "wasn't supported".

Reply:
That's much better then what I had, thanks!